### PR TITLE
Caulie cheese macroni.md

### PR DESCRIPTION
**Pull request by Kush2798**

### CODE REVIEW

File: recipes/Cauli cheese macaroni [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Ingredients
```
Comment (k88hudson) [inline recipes/Cauli cheese macaroni]: Can you add the title above this?

**Review by k88hudson — CHANGES_REQUESTED** [review]
Just add the title and it is good to go!

File: recipes/Cauli cheese macaroni.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+1. Heat the grill to its highest setting and bring a large pan of salted water to the boil. 
+2. Cook the pasta for 4 mins, add the cauliflower for a further 8 mins.
+3. Drain, reserving 100ml of the cooking water.
+4. Melt the butter in a medium pan over a low heat and stir in the flour. 
+5. Slowly pour in the milk, stirring constantly until it begins to thicken. 
+6. Bubble for a few mins, then remove from the heat.
+7. Add the cheese, the reserved cooking water and seasoning to taste.
+8. Tip the pasta and cauliflower into a large casserole dish and stir through the cheese sauce. 
+9. Mix together the garlic, breadcrumbs and thyme, and scatter over the top.
+10.Grill for 2-3 mins until bubbling.
```
Comment (k88hudson) [inline recipes/Cauli cheese macaroni.md]: Can you add a space after the `.`?

File: recipes/Cauli cheese macaroni.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Cauli Cheese Macroni
```
Comment (k88hudson) [inline recipes/Cauli cheese macaroni.md]: Can you change this to `# Cauli Cheese Macaroni`?

**Review by k88hudson — CHANGES_REQUESTED** [review]

**Comment by titaniumbones** [comment]
I see one other small issue -- can you add a space between 10. and the instruction?

**Comment by k88hudson** [comment]
🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆 🎆

**Pull request merged**